contents/docs/self-host/deploy/configuration.md

```diff
@@ -122,53 +122,81 @@ plugins:
-  - Essentially if you know the node that you want ClickHouse to be installed on you can run `kubectl label nodes <desired-clickhouse-node-name> clickhouse=true`
-- Set `.Values.clickhouse.nodeSelector.clickhouse: "true"` in your `values.yaml`. You can find more information about optional settings like that [here](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/values.yaml) and also more about `nodeSelectors` [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
-
-
-### [ClickHouse](https://clickhouse.tech/)
+### [ClickHouse](../runbook/clickhouse/)
 
 ClickHouse is the database that does the bulk of heavy lifting with regards to storing and analyzing the analytics data.
 
 By default, ClickHouse is installed as a part of the chart, powered by [clickhouse-operator](https://github.com/Altinity/clickhouse-operator/). As such it's important to set the database size to be enough to store the raw data via `clickhouseOperator.size` value.
 
 To use an external `ClickHouse` cluster, set `clickhouseOperator.enabled` to `false` and set `clickhouse.host`, `clickhouse.database`, `clickhouse.user` and `clickhouse.password`.
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) for full configuration options._
 
-### [PostgreSQL](https://www.postgresql.org/)
+### [PostgreSQL](../runbook/postgresql/)
 
 > While ClickHouse powers the bulk of the analytics if you deploy PostHog using this chart, Postgres is still needed as a data store for PostHog to work.
 
 By default, PostgreSQL is installed as part of the chart. To use an external PostgreSQL server set `postgresql.enabled` to `false` and then set `postgresql.postgresHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUsername` and `postgresql.postgresqlPort`) may also want changing from their default values.
 
 To avoid issues when upgrading this chart, provide `postgresql.postgresqlPassword` for subsequent upgrades. This is due to an issue in the PostgreSQL chart where password will be overwritten with randomly generated passwords otherwise. See [PostgreSQL#upgrade](https://github.com/helm/charts/tree/master/stable/postgresql#upgrade) for more detail.
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) and [PostgreSQL chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) for full configuration options._
```

Comment:
I don't think we should remove these links. You will need to check our values file + the bitnami docs to get the full picture of all options in $ESOTERIC_USECASE.

Same goes for kafka and redis below.

Reply:
Maybe the solution is to refer to them in ALL_VALUES.md? Let's just not drop them.

Reply:
It was very hard previously to get $NON_STANDARD_USECASE working. By dropping references ALL_VALUES.md + the subchart configuration links we effectively make it impossible for anyone to work out $NON_STANDARD_USECASE without our help. 

It's arguable, but I would consider it a bad thing.

There are ways around this - we could have a section dedicated to these links and disclaimers, but we end up duplicating the sections we already have here.

Reply:
Restored